site/_components/item-grid.njk

```diff
@@ -10,14 +10,23 @@
     @param {number} modifiers.shapeOn: Index of the nth item of the grid that will have a shape in the background.
         A value equals to 0 will turn off the shape. Defaults to 0.
     @param {string} modifiers.variant: If 'icon', it will render the icon-card. if 'subnav' it will render card-subnav. if 'community' it will render external-community.
+    @param {string} modifiers.subtitle: Text for a subtitle beneath the primary title.
 
     @param {string} locale: Code for the current locale, used when 'modifiers.variant' is 'subnav'
 
     @param {string} defaultText: Default text used when 'modifiers.variant' is 'icon'.
  #}-->
 {% from 'external-community.njk' import externalCommunity %} {% from 'card-subnav.njk' import cardSubnav %} {% from 'icon-card.njk' import iconCard %} {% from 'card.njk' import card %} {% macro itemGrid(title, items, modifiers, locale, defaultText) %} {% if items.length > 0 %}
 <section class="item-grid {{modifiers.custom}} {{'item-grid--icon-card' if modifiers.variant == 'icon'}} {{ 'item-grid--subnav' if modifiers.variant == 'subnav' }} {{'item-grid--community' if modifiers.variant == 'community'}} {{'item-grid--circled' if modifiers.shapeOn }}">
-  <h3 class="item-grid__title type--h3">{{title}}</h3>
+  <h3 class="item-grid__title type--h1">{{title}}</h3>
```

Comment:
This seems to conflict with the changes proposed in #310 (https://github.com/chromeos/chromeos.dev/pull/310/files#r822847092)